src/main/database.ts

```diff
@@ -0,0 +1,31 @@
+import {CGDatabase} from "../renderer/services/db/api";
+// @ts-ignore
+import {LevelDbController} from "../renderer/services/db/controller";
+import {config} from "@chainsafe/eth2.0-config/lib/presets/minimal";
+import {IService} from "../renderer/services/interfaces";
+import {CGAccount} from "../renderer/models/account";
+
+export class DatabaseHandler implements IService {
+    private database: CGDatabase;
+
+    public constructor() {
+        this.database = new CGDatabase({config, controller: new LevelDbController({name: "test-level.db"})});
+    }
+
+    public async start(): Promise<void> {
+        await this.database.start();
+    }
+
+    public async stop(): Promise<void> {
+        await this.database.stop();
+    }
+
+    public saveToDatabase(id: string, input: any): void {
+        this.database.account.set(id, input);
+    }
+
+    public async getFromDatabase(id: string): Promise<CGAccount | null> {
+        const account = await this.database.account.get(id);
+        return account;
+    }
```

Comment:
those two methods are not ok.
I would suggest something like 
```javascript
start() {
   await database.start();
   ipcMain.on(DATABASE_SAVE_EVENT, this.handleSaveEvent)
}

stop() {
   await database.stop();
   ipcMain.removeListener(DATABASE_SAVE_EVENT, this.handleSaveEvent)
}

private handleSaveEvent = (event, args: {table, id, serializedData}) => {
   this.database[table].save(id, serializedData);
}
```

Something along these lines. Might be good idea to leave CGDatabase and keep just DbController in main. In that case CGDatabase would handle serialization and just send stuff to database controller over ipc.

That said, ignore this for now, I'll open separate issue that we can deal with after we merge this.